go/arrow/array/string_test.go

```diff
@@ -95,3 +96,43 @@ func TestStringArray(t *testing.T) {
 		t.Fatalf("got=%q, want=%q", got, want)
 	}
 }
+
+func TestStringArray_Empty(t *testing.T) {
```

Comment:
I think we should rename this test to `TestStringBuilder_Empty`.

Reply:
done.